packages/arm-core/src/components/DrawerOverlay/DrawerPanelHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { Close } from '@amsterdam/asc-assets'
+import { Button } from '@amsterdam/asc-ui'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components'
+
+const HeaderContainer = styled.div`
+  display: flex;
+`
+
+const HeaderContent = styled.div``
```

Comment:
Using an empty div as style? Is this necessary?

Reply:
Not really, I had some styles there before. I'll use a plain div.